sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- Removed `Dynamic Classification` feature.
```

Comment:
I think it's safe to just remove all traces of it from the changelog since no customers ever used it.

Reply:
In JS, we're keeping an entry in changelog, but we don't have any preference on it. Want to see what other languages are doing and unify on this.

Reply:
Talked to Java architect, we would not modify the previously released changelog but adding the note at latest beta version.
updated note
"Removed 'Dynamic Classification' feature, which was introduced in the previous beta release"